src/lib.rs

```diff
@@ -214,6 +215,35 @@ impl Database {
         });
     }
 
+    /// Loads a font from the given source into the `Database` and returns
+    /// the ID of the loaded font.
+    pub fn load_font_source_ids(&mut self, source: Source) -> impl Iterator<Item = ID> {
+        let ids = source.with_data(|data| {
+            let n = ttf_parser::fonts_in_collection(data).unwrap_or(1);
+            let mut ids = tinyvec::TinyVec::<[ID; 3]>::with_capacity(
+                n.try_into().expect("Too many fonts in collection"),
```

Comment:
No need for this. `as usize` should be just fine. We don't care about 16bit targets.